README.md

```diff
@@ -190,3 +190,46 @@ What you will see:
 In node3's folder, you should see 8 shards created in the shards folder and a single file generated in the manifest folder. You should then notice that the shards are distributed across nodes 1 and 2. Specifically, the shards in node3 should be found in the `hosted` folders of nodes 1 and 2.
 
 You can then uncomment the `retrieveFile` line in node3.js, but replace the manifest filename with the name of the manifest generated when node3 executed `uploadFile`
+
+#### Demo for CLI sample:
+1. Same steps for node1 & node2 in above "3. Distribute shards to multiple server nodes (batnodes use kadnodes to locate viable hosts)".
+
+   In the third terminal window:
+    1. `cd kad-bat-plugin/node3`(system can't verify correct file path if you don't go to the client's directory)
+    2. `rm -rf dbbb`
+    3. `rm manifest/*`
+    4. `rm shards/*`
+    5. `node clinode3.js`
+    
+2. Open another(4th) terminal window, select the options to upload/download files while connecting to node1
+  - `batchain-sample -u <filePath>`:
```

Comment:
This should be `batchain sample` not `batchain-sample`, right?

Reply:
Actually both will work now--I think that `batchain-sample` is better for local demo since we will need the `batchain` command eventually in our final version. With `batchain-sample`, it gives more confidence that we will not mix up the command results in the future when we have the remote nodes working.

Reply:
👍